File: src/main/java/com/linkhogar/application/house/getById/TenantProfileResponse.java
package com.linkhogar.application.house.getById;

import com.linkhogar.domain.room.enums.Occupation;
import com.linkhogar.domain.user.enums.Gender;

public record TenantProfileResponse(
        Gender gender,
        String ageRange,
        Occupation occupation,
        String description,
        Boolean isSmoker,
        Boolean hasPets
) {}

File: linkhogar-web/src/app/Models/Houses/TenantProfileResponse.ts
// Tipos literales que simulan tus Enums de Java
export type Gender = 'MALE' | 'FEMALE' | 'OTHER';
export type Occupation = 'STUDENT' | 'WORKER' | 'STUDY_AND_WORK' | 'NOT_DEFINED';

export interface TenantProfileResponse {
  gender: Gender | null;
  ageRange: string | null;
  occupation: Occupation | null;
  description: string | null;
  isSmoker: boolean | null;
  hasPets: boolean | null;
}
